### PR TITLE
Sort and group context menu actions

### PR DESCRIPTION
**Pull request by norascheuch**
To make the `select` action more prominent and give the menu more vividness we want to order and group the actions.

image
image

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/package.json [hunk 1]
```diff
@@ -798,19 +798,23 @@
       "view/item/context": [
         {
           "command": "codeQLVariantAnalysisRepositories.removeItemContextMenu",
-          "when": "view == codeQLVariantAnalysisRepositories && viewItem =~ /canBeRemoved/"
+          "when": "view == codeQLVariantAnalysisRepositories && viewItem =~ /canBeRemoved/",
+          "group": "2_qlContextMenu@3"
```
Comment (charisk) [inline extensions/ql-vscode/package.json:802]: Can we give the groups names?

Comment (norascheuch) [reply extensions/ql-vscode/package.json:802]: I guess, I went with naming them similar to the groups we already have.. but there's not really a reason to be cryptic :D

Comment (charisk) [reply extensions/ql-vscode/package.json:802]: Just something a bit less random?

Comment (charisk) [reply extensions/ql-vscode/package.json:802]: Ignore me, just read the docs!

**Review by charisk — APPROVED** [review]

**Pull request merged**